src/blocks/image-gallery/render.js

```diff
@@ -34,15 +40,22 @@ const Render = ( props ) => {
 			) }
 		>
 			{
-				readyToRender
-				? (
-					<ImageGallery
-						attributes={ attributes }
-						setAttributes={ setAttributes }
-						name={ name }
-					/>
-				)
-				: (
+				readyToRender ? (
+					<>
+						<ImageGallery
+							attributes={ attributes }
+							setAttributes={ setAttributes }
+							name={ name }
+						/>
```

Comment:
```suggestion
						<ImageGallery
								{ ...{ name,setAttributes,attributes }}
						/>
```
Can we change like this

Reply:
An update has been done as per this suggestion.